.travis.yml

```diff
@@ -1,5 +1,13 @@
 language: cpp
 
+install:
```

Comment:
Yes, this was intended for cases where Node.js is used for tooling (and it is used somewhere, I remember there were some Node.js-based utils, better have them use [the modern LTS version 4 if not the latest LTS version which is 6](https://github.com/nodejs/LTS#lts-schedule1)).

I could move to that single test if you want, but that means everything else will run a totally ancient version of Node. I can't even understand how Travis has a Node.js instance on a `cpp` box:
- `cpp` https://github.com/travis-ci/travis-build/blob/8067d586b03d5cde21b034e6fbcfa561f5c66748/lib/travis/build/script/cpp.rb#L5 
- `node_js` https://github.com/travis-ci/travis-build/blob/8067d586b03d5cde21b034e6fbcfa561f5c66748/lib/travis/build/script/node_js.rb#L5